LFS171x/fabric-material/chaincode/sample-chaincode.go

```diff
@@ -3,6 +3,10 @@
 // Import dependencies
 // Import the chaincode shim package and the peer protobuf package
 
+/*  Parts of this code was contributed by the Hyperledger Team 
```

Comment:
Hyperledger --> Hyperledger Fabric

Reply:
In addition, instead of contributed, we should use same phrasing that you used in the README. Something like "This code is based on code written by the Hyperledger Fabric community."

Reply:
done